rest-api-spec/test/cluster.put_settings/10_basic.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 setup:
   - skip:
-      version: 0 - 999
+      version:  " - "
```

Comment:
@karmi If you want a special `all` that is doable. I would rather have that then yet another flag like `pending`.